.travis.yml

```diff
@@ -6,9 +6,9 @@ go:
 
 env:
   global:
-   - ARCH=$(go env GOARCH)
-   - TAG="ci"
-   - CODECOV_TOKEN="d1e39bea-f800-45de-8266-3e9b85e2594a"
+    - ARCH=$(go env GOARCH)
+    - TAG="ci"
+    - CODECOV_TOKEN="d1e39bea-f800-45de-8266-3e9b85e2594a"
```

Comment:
Should this be in GitHub secrets too?

Reply:
This isn't a Github workflow, this is the existing Travis CI workflow without changes other than fixing the indentation.